### DISCUSSION TITLE
Add event prefix by default

### DISCUSSION THREAD

**Discussion by mikitahimpel** [nolimits4web/swiper#7066] [Ideas]
Swiper dispatch ‘resize’ event if swiper-container was resized. The 'resize' event bubble to window because in onAny method swiper create instance of CustomEvent with bubble true. By web specification the ‘resize’ event on window should be called only if size of window was changed. But swiper dispatch 'resize' on container resize and this event bubble to window. The behaviour that different from web standard can lead to unexpected things because all libs focus on the standard. Thus unexpected ‘resize’ event of window can lead to bugs and unnecessary rerenders in app.
This issue as @nolimits4web said in https://github.com/nolimits4web/swiper/issues/7030#issue-1894234716 PR can be fixed by adding event-prefix but It will take a lot of time for the user of lib to understand that he needs to add this prefix, because the behaviour similar to what I've seen is not obvious enough, also it may affect the performance in some applications, while developers will not notice it. The fact that this library can colise event names can lead to very non-obvious bugs, and I think it would be better if the event prefix will be used by default.

My issue with reproduction link: https://github.com/nolimits4web/swiper/issues/7030#issue-1894234716 .

**Comment by nolimits4web** [comment]
As it will be a breaking change, it will be set to not empty value in next major update soon
